### ISSUE TITLE
New "process start date" and "process end date" data items

### ISSUE THREAD

**Issue by ddooley** [obi-ontology/obi#1041]
Currenty OBI has a "date process started" [http://purl.obolibrary.org/obo/OBI_0002471] measurement datum, which carries semantics of being about a process that has started in the past.  A few open issues: #898 and #899 have suggested "has start time" "has end time" as data properties, but this approach, though it allows for future time points to be expressed, depends on expanding the list of OBI data properties, which many are averse to. (Is EuPath using these?). This would also help close #981 .

Can we have a more abstract "process start date" data item, which may reference a past or future (intended) date?  Similarly, a "process end date" to capture real or intended completion date.  Process duration can then be calculated from these, or two of the three can be used to calculate the remaining dependency. (Specific values can be attached to instances of these via 'has representation' or 'has value specification' as they arise).  I'd like the definitions to embrace past observations but also simulations and predictions. 

label: 'process start date'
definition: 'A data item that is the date when a process was initiated or is forecast to be initiated'
proposed parent: 'data item' 

Likewise,

label: 'process end date'
definition: 'A data item that is the date when a process was ended or is forecast to end'

(Later we could introduce a 'time datum' class to put these, and age, and other classes of time datums in.)

**Comment by cstoeckert** [comment]
Discussed on OBI call July 15, 2019. Would like to see logical definition started. How does this relate to date of birth?
If the data item is to apply to something to be initiated then need to relabel. Process start date. Process end date. 
Should look at RO and how this is addressed. Annotation property: 'has start time value' and 'has end time value'

**Comment by turbomam** [comment]
We could include the temporal examples at http://oupr.inference.me:8080/ in our analysis.

**Comment by ddooley** [comment]
Here's a diagram of the axiomization that seems to fit with BFO ISO.

![image](https://user-images.githubusercontent.com/4000582/64544553-0df04900-d2dc-11e9-9502-c6cdac25be7a.png)

**Comment by ddooley** [comment]
And here's a diagram of birth day and age datum.  "postnatal age datum" could be called "age since birth datum".  There are all sorts of _OBI (new)_ terms shown there to support a full axiomatization of "postnatal age datum".  I put them in there just to show where this could go in the context of significant and anticipated prenatal and natal dates too.  

![image](https://user-images.githubusercontent.com/4000582/64545018-e188fc80-d2dc-11e9-9f2b-08319efa8989.png)

**Comment by beckyjackson** [comment]
Discussed on OBI call 9/9/2019. The next step is to determine which terms from these diagrams that would be useful to add to OBI.

**Comment by ddooley** [comment]
A follow-up diagram based on today's discussion. Name changed to "age since birth datum"; addition of "aging process", revising "postnatal history" to "postnatal process".

![image](https://user-images.githubusercontent.com/4000582/64558927-e90ace80-d2f9-11e9-831b-1f9abe0401cf.png)

**Comment by ddooley** [comment]
I think we can close this.  Process start/end times can be handled by time ontology.

**Comment by turbomam** [comment]
Closing per @ddooley's Nov 2023 comment: process start/end times can be handled by the time ontology.

**Comment by turbomam** [comment]
Reopening. On review, Damion's 2023 suggestion to close was not substantiated -- there is no specific OBO Foundry time ontology that provides process start/end date data items. OBI has OBI:0002471 ("date process started") but no corresponding "date process ended," which was half the request. The 2019 axiomatization diagrams were never followed up on.

Leaving open as unresolved rather than closing on an unverified claim.

**Comment by ddooley** [comment]
Indeed, OBO / BFO still doesn't seem to have a position on whether or not OWLTime can be used in conjunction with it.  I have been advocating that a BFO process be recognized as a subclass of time:temporalEntity, and therefore inheriting "The properties [:hasBeginning](https://www.w3.org/TR/owl-time/#time:hasBeginning), [:hasEnd](https://www.w3.org/TR/owl-time/#time:hasEnd) and [:hasTemporalDuration](https://www.w3.org/TR/owl-time/#time:hasTemporalDuration) (or its sub-properties)", but so far no progress on that idea.
